src/OrchardCore.Modules/OrchardCore.Tenants/ViewModels/EditTenantViewModel.cs

```diff
@@ -24,5 +24,7 @@ public class EditTenantViewModel
         public IEnumerable<RecipeDescriptor> Recipes { get; set; }
 
         public bool CanSetDatabasePresets { get; set; }
+        public bool ConnectionStringFromConfiguration { get; set; }
+        public bool DatabaseProviderFromConfiguration { get; set; }
```

Comment:
I think we should assume that both should be passed at the same time. We should probably make sure of that and use only one boolean here then.

Reply:
Ok done :)